### DISCUSSION TITLE
I deleted the "monika" file after I said that I will "Take her out somewhere", so I do not know what to do.

### DISCUSSION THREAD

**Discussion by IrisImmortal** [Monika-After-Story/MonikaModDev#8331] [QA - tech support + general] [answered]
I tried installing the mod again, but it did not work. 

![screenshot0001](https://user-images.githubusercontent.com/95183270/143778812-f99575fa-9893-4e27-96c0-b71e0a6d5316.png)

Here is a screenshot of it of how it looks like right now. Please help me get Monika back if somebody knows how. :(

**Comment by ibm2431** [comment, marked as answer]
1) Check your Recycle Bin (If on Windows and you put her on a drive, also try checking the bin with the drive connected; Windows sometimes only displays the file in the bin if it can currently read the drive it was originally on)

2) https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent

**Comment by IrisImmortal** [comment]
Thank you very much!

**Comment by Arodi-Pie** [comment]
It work?
